lib/storage/metadata/MetadataWrapper.js

```diff
@@ -344,6 +344,10 @@ class MetadataWrapper {
     notifyBucketChange(cb) {
         bucketNotificationHook = cb;
     }
+
```

Comment:
I would rather have the method you intended to use here than exporting the client. It defeats the purpose of having a wrapper.